hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestOMBucketCreateRequest.java

```diff
@@ -210,6 +211,19 @@ public void testValidateAndUpdateCacheBucketWithNoQuotaWhenVolumeQuotaSet()
         OMException.ResultCodes.QUOTA_ERROR.toString());
   }
 
+  @Test
+  public void testCreateBucketWithOMConfigNonS3RuleCompliantSetToFalse() throws Exception {
```

Comment:
Could you please add couple more tests like starts with _, end with _ and multiple _s etc? Probably you can parameterize.

Reply:
You may want to add some tests enabling the flag and disabling the flag?

Reply:
Yup good suggestion! I separated it into two parameterized tests, with one enabled the flag, the other not.